### DISCUSSION TITLE
Best Way to Apply System Config Files

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [twpayne/chezmoi#3768] [Q&A]
Hey, I'd like to keep two different `.chezmoiroot` type folders in my git repository. One for the $HOME environment which I have already implemented. I'd also like to be able to run `sudo chezmoi apply` to apply configurations stored in a folder in my Chezmoi project that are in a folder called `system/`. Is there a way I can manually override the `.chezmoiroot` file? Also, is there a way I can apply the project stored in `$HOME/.local/share/chezmoi` to the `/` root path of the system? I'm looking for something like `sudo chezmoi apply --chezmoiroot system --target /`

Thanks in advance!

**Comment by Lockszmith-GH** [comment]
It's very late on my end, so just pointing you at past discussions of similar solutions.

#2574 and [the resolution of #2375](https://github.com/twpayne/chezmoi/discussions/2375#discussioncomment-3735286)
hope this gets you on the right track.

**Reply by ProfessorManhattan** [reply]
Hey @Lockszmith-GH -- hmm.. I checked those out but I'm still unsure on how to use Chezmoi to manage system config files like the ones stored in `/usr/local/etc`

**Reply by halostatue** [reply]
You would have to have a chezmoi configuration that treats `/` as `$HOME` and manages configurations from there *as root*. Chezmoi is explicitly not intended to manage system config files, but personal config files.

**Reply by Lockszmith-GH** [reply]
Exactly what @halostatue stated.

My suggestion was about separating 
the chezmoi context for the system from your user's home. Especially because the system files will need sudo to manipulate them.
